pages/event-resources.jsx

```diff
@@ -0,0 +1,407 @@
+var React = require('react');
+var ImageTag = require('../components/imagetag.jsx');
+
+var LinksContainer = React.createClass({
+  render: function() {
+    return (
+      <div className="row">{this.props.children}</div>
+    );
```

Comment:
This seems like a really trivial React class, do you think it should just be inlined into whatever's using it, or do you think it will eventually become more complex/useful?